src/io/anuke/corebot/Messages.java

```diff
@@ -72,7 +72,7 @@ public Messages(){
                     if(!result.valid){
                         builder.append(Strings.format("```diff\n{0}\n- offline```", result.ip));
                     }else{
-                        builder.append(Strings.format("```http\n{0}\n\nPlayers: {1}\nMap: {2}\nWave: {3}\nVersion: {4}\nPing: {5}ms```",
+                        builder.append(Strings.format("```http\n{0}\n\nHost: {1}\nPlayers: {2}\nMap: {3}\nWave: {4}\nVersion: {5}\nPing: {6}ms```",
```

Comment:
You added a host argument to the string without adding the host name to the parameters. This will have the effect of shifting/scrambling up the message without adding the result's host name.